src/pages/FriendsList.js

```diff
@@ -0,0 +1,69 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import users from '../data/user';
+import TopBar from '../components/TopBar';
+import SearchBox from '../components/SearchBox';
+function FriendsList() {
+  const [searchClick, setSearchClick] = useState(false);
+  const [search, setSearch] = useState('');
+  const onSearchButtonClicked = () => {
+    setSearchClick(!searchClick);
+  };
+  const handleSearchInputChange = (e) => {
+    setSearch(e.target.value);
+  };
+  const userSearchResult = users.filter((user) => {
+    return user.name.includes(search);
+  });
+  return (
+    <Container>
+      <TopBar current="friends" onSearchButtonClicked={onSearchButtonClicked} />
+      {searchClick ? (
+        <SearchBox
+          searchClick={searchClick}
+          setSearchClick={setSearchClick}
+          handleInputChange={handleSearchInputChange}
+        />
+      ) : (
+        ''
+      )}
+      {userSearchResult.map((user) => (
+        <Wrapper key={user.id}>
+          <Img src={process.env.PUBLIC_URL + '/img/' + user.profileImg}></Img>
```

Comment:
이 컴포넌트가 "개별 친구 프로필" 아이템이 아니고 "친구 목록" 컴포넌트에 속하기 때문에 어떤 이미지인지를 컴포넌트 이름을 통해 명시하는게 더 좋을수도 있다고 생각이 들어요
```suggestion
          <FriendProfileImg src={process.env.PUBLIC_URL + '/img/' + user.profileImg}/>
```